include/odp/api/spec/ml_types.h

```diff
@@ -512,6 +512,37 @@ typedef struct odp_ml_shape_info_t {
 
 } odp_ml_shape_info_t;
 
+/**
+ * Quantization parameters
+ *
+ * These parameters are used to convert between floating point and integer data. Scale and zerop
+ * values can be used directly with the odp_ml_fp32_from_*() and odp_ml_fp32_to_*() functions.
+ */
+typedef struct odp_ml_quant_param_t {
+	/**
+	 * Type of quantization scale value
+	 *
+	 * Valid quantization scale and zero point values are provided, if set to something other
+	 * than #ODP_ML_DATA_TYPE_NONE. Allowed types are #ODP_ML_DATA_TYPE_NONE and
+	 * #ODP_ML_DATA_TYPE_FP32.
+	 */
+	odp_ml_data_type_t type;
+
+	/** Quantization scale */
+	float scale_fp32;
+
+	/** Quantization zero point */
+	int32_t zerop_i32;
+
+} odp_ml_quant_param_t;
+
+/** Quantization information */
+typedef struct odp_ml_quant_info_t {
+	/** Quantization parameters common to all input / output data */
+	odp_ml_quant_param_t common;
```

Comment:
Suggesting to change wording to be more clear that it's not common to all data of all inputs (outputs), but all data of one input (output):
"... common to all input / output data" -> " ... common to all data values of an input / output"